components/00-design-tokens/fonts.twig

```diff
@@ -14,12 +14,29 @@
           <p class="text-tiny">A B C D E F G H I J K L M N O P Q R S T U V W X Y Z</p>
           <p class="text-tiny">a b c d e f g h i j k l m n o p q r s t u v w x y z</p>
           <p class="text-tiny">0 1 2 3 4 5 6 7 8 9</p>
+          <div class="weights-styles">
```

Comment:
Do we use all of these weights?

Reply:
we do not, i overdid it.  according to lauren and this: https://www.figma.com/file/Y5cAnf9q3OyAp5q199lWqz6Y/Text-styles?node-id=9%3A0, we're currently using 300, 400, 500.